### PR TITLE
Add Node Messages API snippets

### PR DESCRIPTION
**Pull request by marklewin**
Adding new Node snippets for Messages API. The specs [are here](https://github.com/Nexmo/code-snippet-specs/tree/master/olympus/messages).

Ticket: https://nexmoinc.atlassian.net/browse/DEVX-1787

**Facebook**

- [ ] Send an Image Message
- [ ] Send a Video Message
- [ ] Send an Audio Message
- [ ] Send a Message Template
- [ ] Send a File Message

**WhatsApp**
- [ ] Send a Message Template (MTM)
- [ ] Send a Media Message Template*
- [ ] Send an Image Message
- [ ] Send a Contact
- [ ] Send a Video Message
- [ ] Send a Link Button*
- [ ] Send a Location
- [ ] Send an Audio Message
- [ ] Send a Quick Reply Button*
- [ ] Send a File Message

Unable to test these (*), due to the limitation of the Sandbox `verify` template

**Viber**
- [ ] Send an Image Message

### CODE REVIEW

**Comment by mheap** [comment]
Should these samples be in the `node-code-snippets` repo?

**Comment by marklewin** [comment]
> Should these samples be in the `node-code-snippets` repo?

I can't find any repo of that name? And Alex pushed his most recent changes here (https://github.com/Nexmo/nexmo-node-code-snippets/pull/86)

**Comment by mheap** [comment]
🤦 We're in the `nexmo-node-code-snippets` repo already! Sorry, I misread this as the NDP repo before

File: .env-example [hunk 1]
```diff
@@ -54,3 +54,21 @@ MESSAGES_APPLICATION_ID=12a3b4cd-a1b2-1ab2-a1b2-a1234bc5d678
 VOICE_CALLBACK_TYPE=tel
 VOICE_CALLBACK_VALUE=447700900002
 VOICE_STATUS_URL=https://example.com/webhooks/status
+
+# Messages
+BASE_URL=https://messages-sandbox.nexmo.com/
+MESSAGES_API_URL=https://messages-sandbox.nexmo.com/v0.1/messages
+FB_SENDER_ID=107083064136738
+FB_RECIPIENT_ID=3881986391874929
```
Comment (slorello89) [inline .env-example]: Is this a real sender id?

File: .env-example [hunk 2]
```diff
@@ -54,3 +54,21 @@ MESSAGES_APPLICATION_ID=12a3b4cd-a1b2-1ab2-a1b2-a1234bc5d678
 VOICE_CALLBACK_TYPE=tel
 VOICE_CALLBACK_VALUE=447700900002
 VOICE_STATUS_URL=https://example.com/webhooks/status
+
+# Messages
+BASE_URL=https://messages-sandbox.nexmo.com/
```
Comment (slorello89) [inline .env-example]: this only needs to be a hostname - it should not have the scheme - at least not in the context in which it's being used

File: .env-example [hunk 3]
```diff
@@ -54,3 +54,21 @@ MESSAGES_APPLICATION_ID=12a3b4cd-a1b2-1ab2-a1b2-a1234bc5d678
...
+MESSAGES_API_URL=https://messages-sandbox.nexmo.com/v0.1/messages
```
Comment (slorello89) [inline .env-example]: this field doesn't appear to be being used.

**Review by slorello89 — CHANGES_REQUESTED** [review]
Couple minor items - working with @pardel to see if I can get an account to test out the more advanced stuff.

File: .env-example (hunk 3 above)
Comment (marklewin) [reply .env-example]: Removed!

File: .env-example (hunk 2 above)
Comment (marklewin) [reply .env-example]: Changed!

File: .env-example (hunk 1 above)
Comment (marklewin) [reply .env-example]: Good spot! 😊  Removed.

**Comment by marklewin** [comment]
> Couple minor items - working with @pardel to see if I can get an account to test out the more advanced stuff.

Thanks @slorello89 . If you are referring to the WhatsApp template examples I think that might be quite hard to do - you would need to set up templates using either your own WhatsApp business account (hard to obtain AFAIK) or get access to the Sandbox templates (which I have found impossible). I've just based these on the `curl` examples we were given at some point, but have been unable to test them because the only Sandbox template available is the `verify` one, which I have documented [here](https://developer.nexmo.com/messages/concepts/messages-api-sandbox#whatsapp-templates-for-use-with-the-messages-api-sandbox).

**Review by slorello89 — APPROVED** [review]
I've tested all of the snippets in this review - looks good to me.

**Pull request merged**